src/style.css

```diff
@@ -0,0 +1,145 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.card {
+  width: 200px;
+  padding: 30px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: box-shadow 0.2s;
+}
+
+.card:hover {
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+}
+
+.card__inner {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.card__image {
+  margin: 0 auto 45px;
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  margin-bottom: 5px;
+}
+
+.card__article {
+  line-height: 13px;
+  font-size: 11px;
+  margin-bottom: 16px;
+}
+
+.card__assesment {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 20px;
+  vertical-align: bottom;
+}
+
+.card__reviews {
+  line-height: 13px;
+  color: #000;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.card__price-text {
+  line-height: 12.89px;
+}
+
+.card__price-cost {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18.75px;
+}
+
+.button {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  color: #fff;
+  font-size: 14px;
+  line-height: 16px;
+  border-radius: 5px;
+  text-transform: uppercase;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+  cursor: pointer;
+  transition: color 0.2s linear,
```

Comment:
sort it https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Reply:
thanks, done!